src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -110,6 +129,42 @@ def get(self, request: Request, organization: Organization) -> Response:
                     data={"detail": "You do not have access to any projects."},
                 )
 
+        visibility = serializer.validated_data.get("visibility")
+        if visibility:
+            org_query = GroupSearchView.objects.filter(
+                organization=organization,
+                visibility=GroupSearchViewVisibility.ORGANIZATION,
+            ).prefetch_related("projects")
+
+            owner_query = GroupSearchView.objects.filter(
+                organization=organization,
+                user_id=request.user.id,
+                visibility=GroupSearchViewVisibility.OWNER,
+            ).prefetch_related("projects")
+
+            param_query_map = {
+                "organization": org_query,
```

Comment:
best to use the pre-defined constants instead of pure strings https://github.com/getsentry/sentry/blob/352c4f86082ce805fd4904a0c8fb8fe262447e1d/src/sentry/models/groupsearchview.py#L33